File: Back/src/main/java/com/homeExchanger/Home_exchanger/repository/ConversationRepository.java
package com.homeExchanger.Home_exchanger.repository;

import com.homeExchanger.Home_exchanger.model.Conversation;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface ConversationRepository extends JpaRepository<Conversation, Long> {

}

File: home-exchange-backend/src/main/kotlin/org/isep/homeexchange/infrastructure/repository/ConversationRepository.kt
package org.isep.homeexchange.infrastructure.repository

import org.isep.homeexchange.infrastructure.dao.ConversationDao
import org.springframework.data.jpa.repository.JpaRepository

interface ConversationRepository : JpaRepository<ConversationDao, Long> {

    fun findAllByUserId(id: Long): List<ConversationDao>
}
